### PR TITLE
Fix broken build and failing tests

### PR DESCRIPTION
**Pull request by ryanbogan**
### Description
Fixes the broken build due to a change in babel.
Fixes unit tests by updating expected snapshot that changed due to upstream changes.
Fixes integration test by adding wait before selecting colors.
Changes Cypress test browser to electron in the CI, because the latest version chrome browser used higher memory/CPU which causes the workflow to fail

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by navneet1v** [comment]
@ryanbogan the CI is still failing.

File: .github/workflows/cypress-workflow.yml [hunk 1]
```diff
@@ -155,8 +155,7 @@ jobs:
         uses: cypress-io/github-action@v5
         with:
           working-directory: OpenSearch-Dashboards/plugins/dashboards-maps
-          command: yarn run cypress run --browser chrome
-          wait-on: 'http://localhost:5601'
+          command: yarn run cypress run --headless --browser chrome
```
Comment (junqiu-lei) [inline .github/workflows/cypress-workflow.yml:158]: The latest version chrome browser used higher memory/CPU which crushes the CI runner, let's switch to use `electron` which worked from my CI test. Also please add note here the reason why we switched.

**Comment by ryanbogan** [comment]
Tests passed for cypress but the upload step after is failing

**Comment by junqiu-lei** [comment]
> Tests passed for cypress but the upload step after is failing

@ryanbogan Let's explicitly add cypress videos enabled https://github.com/junqiu-lei/dashboards-maps/blob/30d0ee2154b1e2629af359c788f66d4436eda68b/cypress.config.js#L4

**Comment by ryanbogan** [comment]
CI failure is environment based on Windows only, and all cypress e2e tests are passing on ubuntu-latest.  I also see a similar failure on [security-analytics](https://github.com/opensearch-project/security-analytics-dashboards-plugin/actions/runs/7201952450/job/19619116554) dashboard plugin, which also runs Windows tests, so this might not be able to be fixed from our plugin.

**Review by junqiu-lei — APPROVED** [review]
Thanks, LGTM

**Review by VijayanB — APPROVED** [review]

**Comment by junqiu-lei** [comment]
@ryanbogan backport failed

**Comment by ryanbogan** [comment]
> @ryanbogan backport failed

One of the classes is not present in 2.x.  I'll raise manual backport

**Pull request merged**